cmake/Modules/ToolchainLibraryFinder.cmake

```diff
@@ -90,8 +90,8 @@ function(ToolchainLibraryFinder)
     if(PACKAGE_VERSION_BINARY_ARGUMENTS AND PACKAGE_BINARY)
```

Comment:
Why does this file even exist?

Reply:
Why are you commenting on merge commits? This is the merge in from master